xbmc/pvr/addons/PVRClient.h

```diff
@@ -305,7 +305,7 @@ class CPVRClientCapabilities
 class CPVRClient : public ADDON::IAddonInstanceHandler
 {
 public:
-  explicit CPVRClient(const ADDON::AddonInfoPtr& addonInfo);
+  explicit CPVRClient(const ADDON::AddonInfoPtr& addonInfo, uint64_t instanceId);
```

Comment:
remove `explicit`

and at all (?) other places instance id type is `uint32_t` - why here `uint64_t`?